pkgs/by-name/ze/zensical/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  fetchPypi,
+  rustPlatform,
+  python3Packages,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zensical";
+  version = "0.0.5";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
fetchFromGitHub is preferred

Reply:
zensical/zensical repo has some pre-publish scripts that clone in zensical/ui and link `dist/` as `src/zensical/templates/`.

I was advised to use `fetchPypi` instead of `"combined"`: https://github.com/NixOS/nixpkgs/pull/460052#discussion_r2508256850

In any case, I should add a comment about it.